pkg/kvm/nvme.go

```diff
@@ -43,7 +44,7 @@ func NewVfiouserSubsystemListener(ctrlrDir string) frontend.SubsystemListener {
 
 func (c *vfiouserSubsystemListener) Params(ctrlr *pb.NvmeController, nqn string) spdk.NvmfSubsystemAddListenerParams {
 	result := spdk.NvmfSubsystemAddListenerParams{}
-	ctrlrDirPath := controllerDirPath(c.ctrlrDir, filepath.Base(ctrlr.Spec.SubsystemNameRef))
+	ctrlrDirPath := controllerDirPath(c.ctrlrDir, filepath.Base(fmt.Sprintf("%d", ctrlr.GetSpec().GetNvmeControllerId())))
```

Comment:
Different controllers on different subsystems can have the same id number. Can we use `nqn` then as dir name?